INSTALL.md

```diff
@@ -200,12 +210,7 @@ the Racket fork of chez scheme.
 ```sh
 git clone git@github.com:racket/ChezScheme.git
 cd ChezScheme
-git submodule init
-git submodule update
-arch=tarm64osx
-./configure --pb
-make ${arch}.bootquick
-./configure --threads
+./configure
```

Comment:
I'm working through these on an `aarch64-darwin` machine right now and the changes to this section raise a couple of questions for me:
1. Did the changes as listed not work?
2. Has it become a default to configure for thread support? If not, we want to leave the `--threads` in there because it is important that Chez has support for threads.

Reply:
I was able to spot the answer to question (2) in the configuration script for ChezScheme. That's a yes. Wouldn't hurt to be explicit, still, but shouldn't break anything not to be.